Sync from 'main' to 'test-remote-action-build'

luigimak opened the pull request:
-Sync from 'https://github.com/WildKernels/OnePlus_KernelSU_SUSFS/commit/72d8783cf36536cddc1ccc0f4f1053e3235759b7'
-Modify workflow to select individual OnePlus models
-Switch to using configs, manifests and action from WildKernels/OnePlus_KernelSU_SUSFS
-Remove release creation functionality

luigimak commented:
test OP11_oos16
https://github.com/luigimak/OnePlus_KernelSU_SUSFS/actions/runs/19991987926

test OP11_oos15
https://github.com/luigimak/OnePlus_KernelSU_SUSFS/actions/runs/19991991172

test OP11_oos14
https://github.com/luigimak/OnePlus_KernelSU_SUSFS/actions/runs/19991994298

luigimak commented:
test OP11_oos16
https://github.com/luigimak/OnePlus_KernelSU_SUSFS/actions/runs/20003000165

fatalcoder524 commented:
Is the cleanup and Build options updated?

coz we added more device support and both yml should have it !

Also some oos 16 and oos 14 builds were added!

fatalcoder524 commented:
Also i cant merge because of conflict.

maybe rebase with this branch and then clear conflicts and push here again.

luigimak committed:
Sync from 'main' to 'test-remote-action-build'

-Sync from 'https://github.com/WildKernels/OnePlus_KernelSU_SUSFS/commit/72d8783cf36536cddc1ccc0f4f1053e3235759b7'
-Modify workflow to select individual OnePlus models
-Switch to using configs, manifests and action from WildKernels/OnePlus_KernelSU_SUSFS
-Remove release creation functionality

luigimak commented:
> Also i cant merge because of conflict.
> 
> maybe rebase with this branch and then clear conflicts and push here again.

ok, it should be fine now

fatalcoder524 commented:
OOS 16 missing OP15 i think.. Please verify device list

fatalcoder524 commented on .github/workflows/clean-up.yml:
```diff
@@ -0,0 +1,1170 @@
+          - android14-6.1
+          - android14-5.15
+          - android13-5.15
+          - android13-5.10
```
android13-5.10 not needed

fatalcoder524 commented on .github/workflows/clean-up.yml:
```diff
@@ -0,0 +1,1170 @@
+        default: 'ALL'
+        options:
+          - ALL
+          # Kernel Versions
```
android16-6.12 must be added
